src/views/AboutSelfCareView.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import { Helmet } from "react-helmet-async";
+import Sections from "../modules/Sections";
+import { Grid } from "@material-ui/core";
+import SectionSelector from "../components/Section/SectionSelector";
+
+const AboutView = () => {
+  const [sections, setSections] = useState([]);
+
+  useEffect(() => {
+    const fetchPageData = async () => {
+      let response = await Sections.read("about_self_care");
+      setSections(response);
+    };
```

Comment:
Does this pass the browser without warnings?

Reply:
yes, no warning in console